cluster_pack/spark/spark_config_builder.py

```diff
@@ -1,15 +1,23 @@
 
 import os
-
+import logging
 import pyspark
 from pyspark.sql import SparkSession
 
+try:
+    from s3fs import S3FileSystem
+except ModuleNotFoundError:
+    pass

```

Comment:
Why do you need to import S3FileSystem if you never use it ?

Reply:
Removed